### PR TITLE
Added Customer Key blurb for review

### PR DESCRIPTION
**Pull request by KCCross**
Included links to techcommunity announcement and team blog at anwar's request.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/security-compliance-overview.md [hunk 1]
```diff
@@ -151,6 +151,19 @@ If you'd like further Teams-specific information on configuring content search,
...
 
+## Customer Key
+
+Microsoft 365 offers an additional layer of encryption on top of service encryption for your content. Using keys you provide, Customer Key encrypts several different types of data in Microsoft Teams. Using Customer Key at the application level, Customer Key encrypts Teams files stored in SharePoint Online. For information, see [Service encryption with Customer Key](https://docs.microsoft.com/microsoft-365/compliance/customer-key-overview). 
+
+Using Customer Key at the tenant level, Customer Key encrypts:
+- Teams chat messages (1:1 chats, group chats, meeting chats and channel conversations)
```
Comment (JohanFreelancer9) [inline Teams/security-compliance-overview.md:159]:
```suggestion
- Teams chat messages (1:1 chats, group chats, meeting chats, and channel conversations)
```

File: Teams/security-compliance-overview.md [hunk 2]
```diff
@@ -151,6 +151,19 @@ If you'd like further Teams-specific information on configuring content search,
...
+- Teams media messages (images, code snippets, videos, wiki images)
```
Comment (JohanFreelancer9) [inline Teams/security-compliance-overview.md:160]:
```suggestion
- Teams media messages (images, code snippets, videos, and wiki images)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - two minor edits, please.

**Comment by anwara-ms** [comment]
I cant edit the file for PR 6460. Could you please indicate that we are currently doing public preview of the feature and interested customers can reach out to us at m365ck@microsoft.com ?

**Pull request merged**